OpenEdXMobile/src/test/java/org/edx/mobile/test/module/DownloadTests.java

```diff
@@ -62,9 +62,9 @@ public void testAddDownload() throws Exception {
         print( "averageProgress=" + averageProgress);
 
         // test remove downloads
-        boolean removed = dm.removeDownload(dmid);
+        boolean removed = dm.removeDownloads(dmid) > 0;
```

Comment:
Instead of testing single video removal, we should remove both videos being downloaded and check the no. of removed videos = 2. For example:

````java
final int removedDownloads = dm.removeDownloads(dmid, dmid2);
assertEquals(2, removedDownloads);
````